config/shared/errors/errors.go

```diff
@@ -54,7 +54,7 @@ var (
 	ErrVfatLabelTooLong          = errors.New("filesystem labels cannot be longer than 11 characters when using vfat")
 	ErrLuksLabelTooLong          = errors.New("luks device labels cannot be longer than 47 characters")
 	ErrLuksNameContainsSlash     = errors.New("device names cannot contain slashes")
-	ErrInvalidLuksVolume         = errors.New("a key-file or clevis configuration must be specified")
```

Comment:
In general, we can't remove error values that have been shipped in releases, since they're part of the ABI.  It's probably fine in this case since the error was only used for an -experimental feature.